Lib/webbrowser.py

```diff
@@ -484,6 +485,12 @@ def register_X_browsers():
 
 # Prefer X browsers if present
 if os.environ.get("DISPLAY"):
+    try:
+        cmd = "xdg-settings get default-web-browser".split()
+        _preferred_browser = subprocess.check_output(cmd).decode().strip()
+    except (FileNotFoundError, subprocess.CalledProcessError):
+        pass
```

Comment:
I'm not sure what to report, otherwise. Passing on the FileNotFoundError feels right to me.

I could drop catching CalledProcessError?